### PR TITLE
Licensing: Move up note on declared vs concluded license disagreement

### PR DESCRIPTION
**Pull request by bact**
- Currently the note about what to do when there is a disagreement between `hasDeclaredLicense` and `hasConcludedLicense` is buried down the bottom of the page
- It is also not very noticeable due to the fact that it has no proper heading, make it looks like it is under the "hasConcludedLicense" section. Which in fact, it is not. The last two paragraphs are meant to be at the "root" level of the description.
- This PR move up the note to the position next to the paragraph that introduces `hasDeclaredLicense` and `hasConcludedLicense` concepts.

### CODE REVIEW

**Comment by goneall** [comment]
@swinslow @zvr - Can you do a quick review - if OK, we'd like to merge

**Comment by goneall** [comment]
@bact - could you update to resolve the merge conflicts?

@swinslow - Please take a look - if you agree, we can merge.

**Comment by bact** [comment]
@goneall  Conflict resolved.

File: model/Licensing/Licensing.md [hunk 1]
```diff
@@ -10,17 +10,24 @@ facilitate compliance with typical license use cases.
 ## Description
 
 The Licensing profile only contains the additional requirement that any
-Software Artifact must have a Relationship of type hasConcludedLicense.
+Software Artifact must have a `Relationship` of type `hasConcludedLicense`.
 
-Classes and Property restrictions are defined in the SimpleLicensingProfile
+Classes and Property restrictions are defined in the `SimpleLicensingProfile`
 (Classes and Properties associated with
 [license expression srings](../../annexes/SPDX-license-expressions.md)) and in
```
Comment (swinslow) [inline model/Licensing/Licensing.md:17]: Not an issue with your PR, but did notice when I was looking at this, "srings" should be "strings".

**Review by swinslow — APPROVED** [review]
LGTM -- I did notice one typo (not from your PR) -- if you want to fix that now as well, great! Otherwise looks good to go as-is.

File: model/Licensing/Licensing.md (hunk 1 above)
Comment (bact) [reply model/Licensing/Licensing.md]: Fixed the typo.

**Review by goneall — APPROVED** [review]
Typo fixed - LGTM

**Pull request merged**